simulation/agents.py

```diff
@@ -101,6 +101,12 @@ def reverse_sigmoid(x: float, k: float = SIGMOID_K, offset: float = SIGMOID_OFFS
     return 1 / (1 + math.exp(k * (x + offset)))
 
 
+def sigmoid(x: float, k: float = SIGMOID_K):
+    # Sigmoid function with y bounded between 0 & 1
+
+    return 2 * 1 / (1 + math.exp(-k * (x / 10))) - 1
```

Comment:
doesn't the formula for sigmoid function take the form 1/(1+exp(-X))? Also what does the denominator '10' here refer to?

Reply:
Is this equation provided by CCC? Or what is the aim we hope to achieve with this equation? At first glance:
- `sigmoid(0)` will give 0
- `sigmoid(9999999) will give ~ 1
- but you need very large `x` value to achieve 1
- looks like `x` =  years, plugging in sigmoid(1) ~ 10%. ,sigmoid(5) ~ 46%
- After 1 (5) year(s) of heat pump awareness campaign only 10% (46%) of people becomes aware of heat pump
- 46% after 5 years feels a bit low, no? Is it realistic?

Reply:
This is on top of the initial heat pump awareness factor. If, say, the HP awareness factor is 0.25, then 25% of agents already are HP aware. If this function is called, an additional ~50% (of the remaining agents that are not already HP aware) will then become aware of HPs in 5 years time.

Reply:
why couldn't you just use the SIGMOID_K to do this though - as that's also a normalising factor. Also where's the requirement sigmoid(x=10)=0.75 come from?

And also the numerator 2, and -1? Strictly speaking this is no longer sigmoid function too - would suggest renaming to something else which this distribution.

I think this function resemble more like a tanh function instead, not sigmoid?

Reply:
The factor of `2` stretched the function along the `y` and `-1` shifts the y-intercept to 0. This is required to bound the function between 0 & 1 along the y axis

Reply:
Now using linear function to define probability of becoming HP aware in HP awareness intervention.